user_scripts/wrangler_by_sample.sh

```diff
@@ -43,7 +43,6 @@ singularity_bindings="-B $project_resources:/opt/project_resources
  -B $output_folder:/opt/analysis
  -B $input_sample_sheet_directory:/opt/input_sample_sheet_directory
  -B $fastq_dir:/opt/data
- -B /home/charlie/projects/MIPTools_wrangler_in_sif/snakemake:/opt/snakemake
```

Comment:
excellent - I see this is where the hardcoded shell script bindings are getting taken back out